### PR TITLE
Migrate to Grabl

### PR DESCRIPTION
**Pull request by vmax**
## What is the goal of this PR?

Integrate building `client-python` with Grabl

## What are the changes implemented in this PR?

Add `.grabl/automation.yml` and remove old CircleCI config

### CODE REVIEW

File: .grabl/automation.yml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        bazel run --define version=$(git rev-parse HEAD) //:deploy-pip -- snapshot
+    test-deployment:
+      filter:
+        owner: graknlabs
+        branch: master
+      machine: graknlabs-ubuntu-20.04
+      type: foreground
+      dependencies: [deploy-pip-snapshot]
+      script: |
+        pyenv global 3.6.10
```
Comment (flyingsilverfin) [inline .grabl/automation.yml:79]: do we still need this @vmax ?

File: .grabl/automation.yml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    branch: master
+  validation:
+    validate-dependencies:
+      machine: graknlabs-ubuntu-20.04
+      script: bazel test //:release-validate-deps --test_output=streamed
+  deployment:
+    deploy-github:
+      machine: graknlabs-ubuntu-20.04
+      script: |
+        pyenv global 3.6.10
```
Comment (flyingsilverfin) [inline .grabl/automation.yml:116]: same here...

File: .grabl/automation.yml (hunk 1 above)
Comment (vmax) [reply .grabl/automation.yml:79]: @flyingsilverfin yes; it wouldn't build on Python 3.8 due to old `grpcio`

Comment (flyingsilverfin) [reply .grabl/automation.yml:79]: right ok!

**Review by flyingsilverfin — APPROVED** [review]

**Review by lolski — APPROVED** [review]

**Comment by lolski** [comment]
@alexjpwalker will add the missing `test-grakn` back

**Pull request merged**